.github/workflows/node.js.yml

```diff
@@ -14,6 +14,10 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
+    - name: Install xsltproc
+      run: |
+        sudo apt-get install xsltproc
+
     - name: Cache
```

Comment:
Installing npm before loading caching feels more natural to me. I would move `Use Node.js ${{ matrix.NODE_VERSION }}` before the cache restoring.